File: app/src/main/java/com/kee0kai/thekey/room/model/SettingPairEntry.java
package com.kee0kai.thekey.room.model;


import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.kee0kai.thekey.room.dao.SettingDao;

@Entity(tableName = SettingDao.TABLE_NAME)
public class SettingPairEntry {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(index = true, name = "id")
    public long id;

    @ColumnInfo(name = "value")
    public String value;


    public SettingPairEntry(long id, String value) {
        this.id = id;
        this.value = value;
    }
}

File: app_mobile/src/main/kotlin/com/github/klee0kai/thekey/app/data/room/entry/SettingPairEntry.kt
package com.github.klee0kai.thekey.app.data.room.entry

import android.os.Parcelable
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.github.klee0kai.thekey.app.data.room.dao.SettingDao
import kotlinx.parcelize.Parcelize

@Parcelize
@Entity(tableName = SettingDao.TABLE_NAME)
data class SettingPairEntry(
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(index = true, name = "id")
    val id: Int,

    @ColumnInfo(name = "value")
    val value: String,
) : Parcelable